src/oci/image/config.rs

```diff
@@ -132,7 +124,7 @@ fn test_parse_config_ok() {
             ]
         }"#;
 
-    let m = parse_config(c);
+    let m: Result<Config, serde_json::Error> = serde_json::from_str(c);
```

Comment:
I believe the test function could return this result type so we could use the question mark operator and do some proper check on config?

Reply:
Indeed, I just updated it.